types/react-draft-wysiwyg/index.d.ts

```diff
@@ -0,0 +1,88 @@
+// Type definitions for react-draft-wysiwyg 1.12
+// Project: https://github.com/jpuri/react-draft-wysiwyg#readme
+// Definitions by: imechZhangLY <https://github.com/imechZhangLY>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from 'react';
+import * as Draft from 'draft-js';
+
+export as namespace Wysiwyg;
+
+declare namespace Wysiwyg {
```

Comment:
The namespace is unnecessary. You can just export everything directly as in `export type SyntheticKeyboardEvent = React.KeyboardEvent<{}>; export type SyntheticEvent = ...;`.